applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis_stage.py

```diff
@@ -16,95 +16,115 @@
 except ImportError:
     KratosMultiphysics.Logger.PrintInfo("EigenSolversApplication", "not imported")
 
+# Importing the base class
```

Comment:
a Few questions:

shouldn't InitializeIO, ExecuteInitialize go in the base class? i guess they are the same for fluid and structure.

PrepareModelPartForSolver is a new function of the Solver API? i mean all solvers will have to provide that?

Reply:
eventually these will go the the base class, yes!

Then it will be easier to do only solver-coupling, what you want

We could do this now, but then we will need another PR for updating the baseclass of the AnalysisStage

The `PrepareModelPartForSolver` was introduced by me for @KratosMultiphysics/optimization because they pass an externally read modelpart. Therefore I separated `ImportModelPart` into `ReadModelPart` and `PrepareModelPartForSolver` (which does the filling of the buffer and the creation of the ComputingModelPart)

Reply:
Ok, this makes sense to me. I guess it should go in the API of the solver